extensions/blocks/contact-form/extensions/advanced-settings.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External Dependencies
+ */
+import { find, filter } from 'lodash';
+import { __ } from '@wordpress/i18n';
+import { addFilter } from '@wordpress/hooks';
+import { InspectorAdvancedControls } from '@wordpress/block-editor';
+import { createHigherOrderComponent } from '@wordpress/compose';
+import { TextControl } from '@wordpress/components';
+
+/**
+ * Internal Dependencies
+ */
+import { childBlocks } from '../index';
+
+const withAdvancedControls = createHigherOrderComponent( BlockEdit => {
+	return props => {
+		const { attributes, setAttributes, isSelected } = props;
+
+		const { id } = attributes;
+
+		const supportedBlocks = filter( childBlocks, block => {
+			if ( block.name === 'field-checkbox-multiple' || block.name === 'field-radio' ) {
```

Comment:
Is there a reason why this doesn't support these two blocks?

Reply:
Added these two blocks back in as support, also based on https://github.com/Automattic/jetpack/pull/15362#issuecomment-618176029